dataikuapi/dss/modelevaluationstore.py

```diff
@@ -0,0 +1,298 @@
+        zone.remove_shared(self)
+
+    def get_usages(self):
+        """
+        Get the recipes referencing this model evaluation store
+
+        Returns:
+            a list of usages
+        """
+        return self.client._perform_json("GET", "/projects/%s/modelevaluationstores/%s/usages" % (self.project_key, self.mes_id))
+
+    def get_object_discussions(self):
+        """
+        Get a handle to manage discussions on the model evaluation store
+
+        :returns: the handle to manage discussions
+        :rtype: :class:`dataikuapi.discussion.DSSObjectDiscussions`
+        """
+        return DSSObjectDiscussions(self.client, self.project_key, "MODEL_EVALUATION_STORE", self.mes_id)
+
+    ########################################################
+    # Deletion
+    ########################################################
+
+    def delete(self):
+        """
+        Delete the model evaluation store
+
+        """
+        return self.client._perform_empty("DELETE", "/projects/%s/modelevaluationstores/%s" % (self.project_key, self.mes_id))
+
+
+    ########################################################
+    # Runs
+    ########################################################
+
+    def list_model_evaluations(self, as_type=None):
+        """
+        List the model evaluations in this model evaluation store.
+
+        :returns: The list of the model evaluations
+        :rtype: list
+        """
+        items = self.client._perform_json("GET", "/projects/%s/modelevaluationstores/%s/runs/" % (self.project_key, self.mes_id))
+        if as_type == "objects" or as_type == "object":
+            return [DSSModelEvaluation(self, item["ref"]["runId"]) for item in items]
+        else:
+            return items
+
+    def get_model_evaluation(self, run_id):
+        """
+        Get a handle to interact with a specific model evaluations
+       
+        :param string run_id: the id of the desired model evaluation
+        
+        :returns: A :class:`dataikuapi.dss.modelevaluationstore.DSSModelEvaluation` model evaluation handle
+        """
+        return DSSModelEvaluation(self, run_id)
+
+    def delete_model_evaluations(self, evaluations):
+        """
+        Remove model evaluations from this store
+        """
+        obj = []
+        for evaluation in evaluations:
+            if isinstance(evaluation, DSSModelEvaluation):
+                obj.append(evaluation.run_id)
+            elif isinstance(evaluation, dict):
+                obj.append(evaluation['run_id'])
+            else:
+                obj.append(evaluation)
+        self.model_evaluation_store.client._perform_json(
+                "DELETE", "/projects/%s/modelevaluationstores/%s/runs/" % (self.project_key, self.mes_id, self.run_id), body=obj)
+
+
+    def create_model_evaluation(self, labels=None, prediction_type=None, model_type=None, model_params=None, data_type=None, data_params=None, metric_params=None, active_classifier_threshold=None):
```

Comment:
Note for later: I'm not exactly sure about the use case, do we want to allow ME creation outside ER/SER ?